mythril/analysis/solver.py

```diff
@@ -192,6 +196,15 @@ def _set_minimisation_constraints(
 
         # Minimize
         minimize.append(transaction.call_data.calldatasize)
-        minimize.append(transaction.call_value)
+
+    for account in world_state.accounts.values():
+        # Lazy way to prevent overflows and to ensure "reasonable" balances
+        # Each account starts with less than 100 ETH
+        constraints.append(
+            UGE(
+                symbol_factory.BitVecVal(100000000000000000000, 256),
```

Comment:
we can skip it if the balance is already concrete. That may probably solve the issue with the evm tests.

Reply:
I made the change, but I wasn't able to revert my change to `evm_tests.py` without the 3 tests failing again. I believe what is happening is that open_states expects that the contract exists, but for those 3 tests, suicide is called in the constructor. So it's possible my change is fine.